runner-review.sh

```diff
@@ -0,0 +1,333 @@
+#!/bin/bash
```

Comment:
Correct , this will solve a lot of issues(like command-not found , etc....) , but the script will stop also in case the input command execution fail that's not good - the input command executions return-code should be handled by the main script . 
So the suggested options here are :-
Options 1 : to disable the -e while command execution , and enable it after it done like the below 
set +e
${command}
set -e

option 2 : to run the command and add an option to success process , then the whole line will return 0 so the script will not exit .
example : ${command} || true ;